### DISCUSSION TITLE
Procedures

### DISCUSSION THREAD

**Discussion by VasileiosGi** [cyrilgdn/terraform-provider-postgresql#313] [Q&A]
Hello! I would like to ask if there is a way to create a procedure. Will this ability provided in the future?

**Comment by AndrewJackson2020** [comment]
Looking through the docs and codebase it looks like procedures are not supported. Unfortunately this is very important for my use case. Would love to see this get added in the future.

**Comment by AndrewJackson2020** [comment]
Just a heads up, opened  PR #440 that adds support for procedures.
